docs/assets/scripts/BookAPI.js

```diff
@@ -76,4 +76,22 @@ function findMovieByTitle(titleText, callback){
             callback(data.Error);
         }
     })   
+}
+
+function findMoviesByTitle(titleText, callback){
+    fetch(movieAPILink + movieAPIKey + "&s=" + titleText)
+    .then(response => {
+        if(response.ok){
+            return response.json();
+        } else {
+            return response.status;
+        }
+    })
+    .then(data => {
+        if(!data.Error){
+            callback(data.Search);
```

Comment:
good first draft; would look to normalize the results into a standard response so the person implementing the site side doesn't have to know the source API (and if you change source APIs they still get the same thing so don't have to worry about changes in the API layer).